src/main/app/form/Form.ts

```diff
@@ -1,15 +1,25 @@
-import { DefinedError, ValidateFunction } from 'ajv';
 
 export class Form<T> {
 
   constructor(
-    private readonly validator: ValidateFunction
+    private readonly validator: any
```

Comment:
There is a type for this... I think it's Form<T> or FormBody<T>

Reply:
Also rename the property to form rather than validator

Reply:
maybe content

Reply:
yh, i forgot